src/plugins/custom_import_map/public/components/custom_vector_upload.tsx

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import React, { useEffect } from 'react';
+import { EuiButton } from '@elastic/eui';
+import { getPlugins } from '../services';
+import EmptyPrompt from './empty_prompt';
+import VectorUploadOptions from './vector_upload_options';
+import { RegionMapOptionsProps } from '../../../../src/plugins/region_map/public';
+
+function CustomVectorUpload(props: RegionMapOptionsProps) {
+  const GEOSPATIAL_PLUGIN = 'opensearch-geospatial';
+  const [isGeospatialPluginInstalled, setIsGespatialPluginInstalled] = React.useState(false);
+
+  // Logic for implementing empty state pattern
```

Comment:
Do we need this anymore?

Reply:
Will get rid of this one in the next PR.